ios/FluentUI/People Picker/MSAvatarView.swift

```diff
@@ -48,6 +48,17 @@ import UIKit
             return CGSize(width: 70, height: 70)
         }
     }
+
+    var squareCornerRadius: CGFloat {
```

Comment:
The other vars in here are public - just wanted to make sure this is internal by design.

Reply:
nit: Would also prefer this new var was documented, even if it is internal.

Reply:
yeah i wasn't really inclined to make this public. but i can document it.